js/understanding-hoisting-the-real-way/understanding-hoisting-the-real-way.md

```diff
@@ -0,0 +1,185 @@
+Hello, I am Shivam Verma. Today we are going to understand one of the most frequently asked questions in interviews. 
+What is **Hoisting** in JavaScript? 
+
+First of all, let us understand the concept of **Hoisting**. Let's Begin
```

Comment:
You made it clear what we are learning in the opening line. You can skip this line, it's not adding any meaning more meaning to it.